src/nadzoring/plugins/connectors/arp.py

```
@@ -0,0 +1,117 @@
+
+            if alert_list:
+                return ProbeResult(
+                    status="error",
+                    error=f"ARP spoofing detected: {len(alert_list)} alert(s)",
+                    details={"data": alert_list, "count": len(alert_list)},
+                )
+            return ProbeResult(
+                status="ok",
+                details={"data": [], "count": 0},
+            )
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)